src/posterior_gp/approx_posterior_gp.jl

```diff
@@ -1,4 +1,25 @@
-struct VFE end
+"""
+    VFE{Tz<:AbstractVector, Tj<:Real}
```

Comment:
```suggestion
    VFE(z::AbstractVector; jitter=default_jitter)
```
Might be an idea to also discuss the `jitter` term in the docstring.